Wikipedia/Code/WMFWelcomeAnimationViewControllers.swift

```diff
@@ -2,11 +2,14 @@ class WMFWelcomeAnimationViewController: UIViewController {
     var welcomePageType:WMFWelcomePageType = .intro
     fileprivate var hasAlreadyAnimated = false
     
-    open lazy var animationView: WMFWelcomeAnimationView? = nil
+    func getAnimationView() -> WMFWelcomeAnimationView? {
```

Comment:
This might be clearer as `setupAnimationView` where subclassers set the `animationView` property